src/controllers/krate/publish.rs

```diff
@@ -114,6 +114,7 @@ pub async fn publish(app: AppState, req: BytesRequest) -> AppResult<Json<GoodCra
             let name = new_crate.name;
             let vers = &*new_crate.vers;
             let links = new_crate.links;
+            let rust_version = new_crate.rust_version.as_deref();
```

Comment:
Should we do some kind of validation that this is a version number and not some random string?

Reply:
This validation already occurs when deserializing the crate information (see [`EncodableRustVersion`](https://github.com/cassaundra/crates.io/blob/427d874653615331fd028fb09929039e23111bc3/src/views/krate_publish.rs#L184-L199)).